src/app/features/public-form/public-form-config.ts

```diff
@@ -1,15 +1,46 @@
 import { Entity } from "../../core/entity/model/entity";
 import { DatabaseEntity } from "../../core/entity/database-entity.decorator";
 import { DatabaseField } from "../../core/entity/database-field.decorator";
+import { LongTextDatatype } from "app/core/basic-datatypes/string/long-text.datatype";
 import { FieldGroup } from "app/core/entity-details/form/field-group";
 
 @DatabaseEntity("PublicFormConfig")
 export class PublicFormConfig extends Entity {
-  @DatabaseField() title: string;
-  @DatabaseField() description: string;
-  @DatabaseField() entity: string;
-  @DatabaseField() columns: FieldGroup[];
+  static override label = $localize`:PublicFormConfig:Public Form`;
+  static override labelPlural = $localize`:PublicFormConfig:Public Forms`;
+  static override route = "admin/public-form";
 
+  @DatabaseField({
+    label: $localize`:PublicFormConfig:Title`,
+  })
+  title: string;
+  @DatabaseField({
```

Comment:
can you keep an empty line between these, please? I find it much easier to read which annotation refers to which line with some more spacing
```suggestion
  title: string;

  @DatabaseField({
```